Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -188,13 +188,14 @@ var ListView = React.createClass({
     /**
      * (sectionData, sectionID) => renderable
      *
-     * If provided, a sticky header is rendered for this section.  The sticky
-     * behavior means that it will scroll with the content at the top of the
-     * section until it reaches the top of the screen, at which point it will
-     * stick to the top until it is pushed off the screen by the next section
-     * header.
+     * If provided, a sticky header / footer is rendered for this section.
+     * The sticky behavior means that it will scroll with the content at the
+     * top or bottom of the section until it reaches the edge of the screen, at
+     * which point it will stick to the top / bottom until it is pushed off
+     * the screen by the next section header / footer.
```

Comment:
FWIW I think it's fine to just add a second note to renderSectionFooter and make the edits there, rather than combine the two methods.